gfx/thebes/gfxPlatform.cpp

```diff
@@ -182,6 +182,10 @@ qcms_transform* gfxPlatform::gCMSInverseRGBTransform = nullptr;
 qcms_transform* gfxPlatform::gCMSRGBATransform = nullptr;
 qcms_transform* gfxPlatform::gCMSBGRATransform = nullptr;
 
+namespace mozilla {
+  extern std::string SignedNumberToStringRecordReplayWorkaroundForWindows(int64_t v);
```

Comment:
Where is `SignedNumberToStringRecordReplayWorkaroundForWindows` defined?

Reply:
In `dom/media/webrtc/jsep/JsepSessionImpl.cpp`.  Probably where it was first needed.  Might want to move it somewhere central eventually.

Reply:
If I run into this again I'll clean it up and move it somewhere proper.